lua/lspconfig/configs/ada_ls.lua

```diff
@@ -0,0 +1,37 @@
+local util = require 'lspconfig.util'
+local bin_name = 'ada_language_server'
+
+if vim.fn.has 'win32' == 1 then
+  bin_name = 'ada_language_server.exe'
+end
```

Comment:
ternary saves some code and is clearer. but i thought that this was handled automatically? @glepnir

Reply:
This might be a remnant from when I first added the config in 2020, perhaps it's not necessary anymore. I'll try to double check.

Reply:
Confirmed that this works fine by default, PR updated. Thanks for catching this.

Reply:
it is no longer needed. we use exepath with server.cmd now see #2595